configs/coins/optimism_dtl.json

```diff
@@ -0,0 +1,36 @@
+{
+  "coin": {
+    "name": "Optimism DTL",
+    "shortcut": "ETH",
+    "label": "Optimism",
+    "alias": "optimism_dtl"
+  },
+  "ports": {
+    "backend_http": 8302
+  },
+  "backend": {
+    "package_name": "backend-optimism-dtl",
+    "package_revision": "satoshilabs-1",
+    "system_user": "optimism",
+    "version": "0.5.50",
+    "binary_url": "https://github.com/ethereum-optimism/optimism/archive/refs/tags/@eth-optimism/data-transport-layer@0.5.50.tar.gz",
+    "verification_type": "sha256",
+    "verification_source": "4f782482d61ef3fa17de6b6a4ad3de441286b4397121e4e6077b2a0392126462",
+    "extract_command": "mkdir backend/source && tar -C backend/source --strip 1 -xf",
+    "exclude_files": [],
+    "exec_command_template": "/bin/sh -c '{{.Env.BackendInstallPath}}/{{.Coin.Alias}}/optimism_dtl_exec.sh 2>> {{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend/{{.Coin.Alias}}.log'",
+    "exec_script": "optimism_dtl.sh",
+    "logrotate_files_template": "{{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend/{{.Coin.Alias}}.log",
+    "postinst_script_template": "cd {{.Env.BackendInstallPath}}/{{.Coin.Alias}}/source && yarn install --frozen-lockfile && yarn build",
```

Comment:
In an effort to keep the deb Dockerfile minimal and not include chain specific dependencies, I set up the `postinst_script_template` as you suggested. It is important to note this strategy now requires local dependencies (node v16 and yarn) to work successfully. I also left the source in tact as it is a node service using lerna workspaces and has a bunch of local aliasing within the project needed to run (didn't seem worth trying to pluck out all of the project dependencies just to trim down the source dir a bit).

Reply:
An alternative that may be over-engineering right now would be building out the make/deb pipeline to support chain specific Dockerfiles to encapsulate the build dependencies without polluting the base deb image.